### DISCUSSION TITLE
Como se llamaba este componente HTML?

### DISCUSSION THREAD

**Discussion by julchat** [dds-jv/foro#161] [Consultas de UI / REST] [answered]
Hola, estamos resolviendo parciales y se nos ocurrio hacer como una secuencia de cards en horizontal, y por si eran mas de las que entraban en una pantalla, que el usuario pueda poner el ratón sobre una flechita en la derecha y que se vaya desplazando con los cards?.

Bueno como se llamaría ese componente HTML que desplaza y supongo que tendrá algo también de front end para poder hacer algo así

Gracias y saludos!

**Comment by gastonprieto** [comment]
Buenas @julchat !

Eso que queres creo que es una barra de desplazamiento (scrollbar) o sino un carousel es lo otro. Recomendación no hagan pantallas que no saben como hacer (en el parcial) :P 

> Bueno como se llamaría ese componente HTML que desplaza y supongo que tendrá algo también de front end para poder hacer algo así

No entiendo a que te referís con esto, HTML describe la parte del front end.

**Comment by julchat** [comment]
Sisi era el carousel de lo que hablaba.

> No entiendo a que te referís con esto, HTML describe la parte del front end.

Justamente a eso voy, carousel segun veo es un elemento de bootstrap. Sería válido detallarlo como elemento HTML?

**Comment by gastonprieto** [comment]
Si pusieras que es ese componente que sería de bootstrap y se resuelve si pones Javascript + bootstrap tal vez si. Sería valido usarlo.

Técnicamente no es un componente HTML nativo, que es en general lo que pedimos que usen en sus pantallas.

**Comment by julchat** [comment, marked as answer]
Entendido. Gracias!
